inc/sainexthopgroup.h

```diff
@@ -56,6 +56,9 @@ typedef enum _sai_next_hop_group_type_t
     /** Next hop group is class-based, with members selected by Forwarding class */
     SAI_NEXT_HOP_GROUP_TYPE_CLASS_BASED,
 
+    /** Next hop hardware protection group. This is the group backing up the primary in the protection group type and is managed by hardware */
+    SAI_NEXT_HOP_GROUP_TYPE_HW_PROTECTION,
```

Comment:
Add details about the condition when a port is carrying traffic for primary path and regular nhg. In this case drops on port will account for both failover drops as well as link drops.